### PR TITLE
Add tags to pkgpanda swagger

### PR DESCRIPTION
**Pull request by karlkfi**
## High Level Description

Add tags to pkgpanda swagger. This makes the endpoints show as grouped in Swagger UI.

screenshot

### CODE REVIEW

**Comment by karlkfi** [comment]
Will be used in the dcos-docs by https://github.com/dcos/dcos-docs/pull/918

File: pkgpanda/docs/http-swagger.yaml [hunk 1]
```diff
@@ -4,6 +4,11 @@ info:
   version: '1.0.0'
 basePath: /pkgpanda
 
+tags:
+  - name: repository
+    description: manage installed packages
+  - name: active
+    description: manage activated packages
```
Comment (branden) [inline pkgpanda/docs/http-swagger.yaml]: Nit: `s/activated/active/` to match the rest of the doc.

Comment (karlkfi) [inline pkgpanda/docs/http-swagger.yaml]: well, it allows you to activate packages that are installed... imo the top level primitives are poorly named, but yeah I can change it...

**Comment by karlkfi** [comment]
rebased to see if the tests will pass (this PR shouldn't affect any tests).

**Pull request merged**